dotcom-rendering/src/components/Standfirst.tsx

```diff
@@ -1,23 +1,16 @@
 import { css } from '@emotion/react';
 import { ArticleDesign, ArticleDisplay, ArticleSpecial } from '@guardian/libs';
-import {
-	from,
-	headline,
-	neutral,
-	space,
-	textSans,
-} from '@guardian/source-foundations';
+import { from, headline, space, textSans } from '@guardian/source-foundations';
 import sanitise from 'sanitize-html';
 import { interactiveLegacyClasses } from '../layouts/lib/interactiveLegacyStyling';
-import { decidePalette } from '../lib/decidePalette';
-import type { Palette } from '../types/palette';
+import { palette } from '../palette';
 
 type Props = {
 	format: ArticleFormat;
 	standfirst: string;
 };
 
-const nestedStyles = (format: ArticleFormat, palette: Palette) => {
+const nestedStyles = (format: ArticleFormat) => {
```

Comment:
Note: I’m wondering whether `offset` should also be a custom property…

Reply:
I wondered about this too or making format available as a template class maybe so we could access via `:has()` - it seems such a shame to drill format through just for this - but both of these also felt a bit magical.